RecoLocalMuon/GEMSegment/plugins/GEMSegmentAlgorithm.cc

```diff
@@ -314,20 +314,26 @@ void GEMSegmentAlgorithm::buildSegments(const GEMEnsemble& ensemble, const Ensem
   proto_segment.clear();
   
   // select hits from the ensemble and sort it
-  const GEMSuperChamber * suCh   = ensemble.first;
+  const GEMEtaPartition * suCh   = ensemble.first;
+  edm::LogVerbatim("GEMSegmentAlgorithm") <<"GEMSegmentAlgorithm::buildSegments error 1"<< std::endl;
   for (auto rh=rechits.begin(); rh!=rechits.end();rh++){
     proto_segment.push_back(*rh);
+  edm::LogVerbatim("GEMSegmentAlgorithm") <<"GEMSegmentAlgorithm::buildSegments error 1.1"<< std::endl;
```

Comment:
the indentation is inconsistent with neighboring lines.

Also, the LogVerbatim is compiled and executed. So, all the string/stream manipulations contribute to the CPU cost, but are not really needed. Please cleanup or put behind if (debug) or change to LogTrace